src/ipa_pc/data_structures.rs

```diff
@@ -947,3 +773,14 @@ impl<F: PrimeField> CanonicalDeserialize for SuccinctCheckPolynomial<F> {
         Ok(SuccinctCheckPolynomial(values))
     }
 }
+
+/// The succinct part of the verifier returns a succinct-check polynomial and final comm key
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct VerifierState<G: Curve> {
```

Comment:
That's basically a DLOGAccumulator we have defined also in ginger-lib. Do we need this duplication here too ?

Reply:
You mean DLogItem, right? If there is no logical difference between these structure I think yes, we should combine it into one. As for me - we should just use this new VerifierState struct from poly-commit instead of separate DLogItem structure, this also simplify the code structure and reduce a bit the total amount of code inside darlin. What do you think?

Reply:
Ok, I've renamed this VerifierState struct to DLogItem